examples/scripts/build.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash -e
```

Comment:
This is not a "valid" shebang, use this instead:
```bash
#!/usr/bin/env bash

set -e
```

Reply:
I think it is haha and it works. but we can change other files on other PRs, this is just merging the examples folder

Reply:
It works, but it goes against good practice

Reply:
we can open a followup pr and you can teach me why :)